src/cryojax/simulator/_optics.py

```diff
@@ -17,214 +16,176 @@
     Constant,
 )
 from ..coordinates import cartesian_to_polar
-from ..typing import Real_, RealImage, Image, ComplexImage, ImageCoords
+from ..typing import Real_, RealImage, ComplexImage, Image, ImageCoords
 
 
 class CTF(AbstractFourierOperator, strict=True):
-    """
-    Compute a Contrast Transfer Function (CTF).
-
-    See ``cryojax.simulator.compute_ctf`` for more
-    information.
-
-    Attributes
-    ----------
-    degrees :
-    defocus_u :
-    defocus_v :
-    defocus_angle :
-    voltage :
-    spherical_aberration :
-    amplitude_contrast_ratio :
-    phase_shift :
+    """Compute the Contrast Transfer Function (CTF) in for a weakly
+    scattering specimen.
+
+    **Attributes:**
+
+    `defocus_u`: The major axis defocus in Angstroms.
+
+    `defocus_v`: The minor axis defocus in Angstroms.
+
+    `astigmatism_angle`: The defocus angle.
+
+    `voltage_in_kv`: The accelerating voltage in kV.
+
+    `spherical_aberration_in_mm`: The spherical aberration coefficient in mm.
+
+    `amplitude_contrast_ratio`: The amplitude contrast ratio.
+
+    `phase_shift`: The additional phase shift.
+
+    `degrees`: Whether or not the `astigmatism_angle` and `phase_shift` are given
+              in degrees or radians.
     """
 
     defocus_u: Real_ = field(default=10000.0, converter=jnp.asarray)
     defocus_v: Real_ = field(default=10000.0, converter=jnp.asarray)
-    defocus_angle: Real_ = field(default=0.0, converter=jnp.asarray)
-    voltage: Real_ = field(default=300.0, converter=jnp.asarray)
-    spherical_aberration: Real_ = field(default=2.7, converter=jnp.asarray)
-    amplitude_contrast: Real_ = field(default=0.1, converter=jnp.asarray)
+    astigmatism_angle: Real_ = field(default=0.0, converter=jnp.asarray)
```

Comment:
could have units here too: `astigmatism_angle_in_rad/deg`

Reply:
I’ve thought about this also… since I’ve added the option everywhere in the package to use degrees or radians with an optional `degrees` boolean, I wasn’t sure a good solution to this

Reply:
Yeah I saw the degrees bool after I wrote this comment. 

Besides checking by assertion, I'm not sure how this could be verified. Might have to rely on user to not introduce bugs.

Reply:
Yes, either this or remove the option all together and stick to degrees… not totally sure but we can stick to this for now.